runtime/cudaq/platform/default/rest/helpers/equal1/Equal1ServerHelper.cpp

```diff
@@ -111,6 +114,11 @@ void Equal1ServerHelper::initialize(BackendConfig config) {
 
   devicePassPipeline = queryPassPipeline();
 
+  if (config.count("emulate") && config["emulate"] == "false")
```

Comment:
what happens if config.count("emulate") equals to 0? 
That means it will assume no emulation instead of not config not existing

Reply:
Basically we assume that if the "emulate" keyword is not present we are not emulating. I'm not sure if the key is required to be present in the config dictionary. This is the reason why I implemented it like this

Reply:
but if the emulate keyword is not present, then we will not get into the if-branch cause count will return 0 right ?

Reply:
yes